<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGeneral</class>
 <widget class="QWidget" name="ConfigureGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string>General</string>
  </property>
  <layout class="QHBoxLayout" name="HorizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="VerticalLayout">
     <item>
      <widget class="QGroupBox" name="GeneralGroupBox">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QHBoxLayout" name="GeneralHorizontalLayout">
        <item>
         <widget class="QWidget" name="general_widget" native="true">
          <layout class="QVBoxLayout" name="GeneralVerticalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="LinuxGroupBox">
       <property name="title">
        <string>Linux</string>
       </property>
       <layout class="QVBoxLayout" name="LinuxVerticalLayout_1">
        <item>
         <widget class="QWidget" name="linux_widget" native="true">
          <layout class="QVBoxLayout" name="LinuxVerticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="DataDirsGroupBox">
       <property name="title">
        <string>Additional Directories (Updates, DLC, etc.)</string>
       </property>
       <layout class="QVBoxLayout" name="DataDirsVerticalLayout">
        <item>
         <widget class="QListWidget" name="external_dirs_list">
          <property name="toolTip">
           <string>This list contains directories that will be searched for game updates and DLC.</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="DataDirsHorizontalLayout">
          <item>
           <spacer name="spacer_dirs_left">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="add_dir_button">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remove_dir_button">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_reset">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="button_reset_defaults">
         <property name="text">
          <string>Reset All Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_reset">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
